packages/custom-field/theme/lumo/vaadin-custom-field-styles.js

```diff
@@ -83,6 +83,11 @@ const customField = css`
   :host([theme~='small'][has-label]) [part='error-message'] {
     font-size: var(--lumo-font-size-xxs);
   }
+
+  /* When custom-field is used without components without outer margin */
```

Comment:
```suggestion
  /* When custom-field is used with components without outer margin */
```
We want to use this if the inner component does not have a margin.

Reply:
done